client/state/posts/test/selectors.js

```diff
@@ -325,6 +326,65 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getSitePostsHierarchyForQueryIgnoringPage()', () => {
+		beforeEach( () => {
+			getSitePostsHierarchyForQueryIgnoringPage.memoizedSelector.cache.clear();
+		} );
+
+		it( 'should return null if the last page is not known', () => {
+			const isLastPage = isSitePostsLastPageForQuery( {
+				posts: {
+					queriesLastPage: {}
+				}
+			}, 2916284, { search: 'Hello' } );
+
+			expect( isLastPage ).to.be.null;
+		} );
+
+		it.only( 'should return a concatenated array of all site posts ignoring page, preserving hierarchy', () => {
```

Comment:
>:red_circle: `it.only`.

Fixed in c3ea05e . Also opened Automattic/eslint-plugin-wpcalypso#2 as an option for a new ESLint rule.